main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/codescalersinternships/Datetime-Client-MohamedFadel/client"
+)
+
+func main() {
+	serverURL := os.Getenv("SERVER_URL")
+	serverPort := os.Getenv("SERVER_PORT")
+	serverPortGin := os.Getenv("SERVER_PORT_GIN")
+
+	if serverURL == "" || serverPort == "" || serverPortGin == "" {
+		log.Fatal("SERVER_URL, SERVER_PORT, and SERVER_PORT_GIN must be set")
+	}
+
+	client.SetupEnv(serverURL, serverPort, serverPortGin)
+
+	serverTypes := []string{"standard", "gin"}
+	contentTypes := []string{"application/json", "text/plain"}
+
+	for _, serverType := range serverTypes {
```

Comment:
I think the client can take an argument representing the response type, and then you can use it to fetch the date and time with the required format so your client can be more generic